### PR TITLE
Bundle size

### PR DESCRIPTION
**Pull request by ankityadav4**
## Related Issue(s)

- Fixes #

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/9ae4e5d426eb1a88f56388f620a4265b1302c664'>9ae4e5d426eb1a88f56388f620a4265b1302c664</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: packages/react-charting/webpack.config.js [hunk 1]
```diff
@@ -1,9 +1,39 @@
...
+                keep_classnames: false,
+                // eslint-disable-next-line @typescript-eslint/naming-convention
+                keep_fnames: false,
+                properties: {
+                  //regex: /^_/,
+                  regex: /((?<=\b|\.)(?=\b[a-zA-Z$_])\w+)|^_/g,
+                },
+              },
+              output: {
+                beautify: false,
```
Comment (AtishayMsft) [inline packages/react-charting/webpack.config.js:29]: beautify option is false by default.
https://terser.org/docs/options/#format-options

File: packages/react-charting/webpack.config.js [hunk 2]
```diff
@@ -1,9 +1,39 @@
...
+          new TerserPlugin({
+            parallel: true,
+            terserOptions: {
+              // https://github.com/terser/terser
+              mangle: {
+                // properties: {
+                //   regex: /^_/,
+                // },
+                // eslint-disable-next-line @typescript-eslint/naming-convention
+                keep_classnames: false,
```
Comment (AtishayMsft) [inline packages/react-charting/webpack.config.js:20]: both these options are false by default. Is it not true for fluent.

Comment (AtishayMsft) [reply packages/react-charting/webpack.config.js:20]: did you try setting toplevel = true

Comment (AtishayMsft) [reply packages/react-charting/webpack.config.js:20]: did you try running 1 more pass of the compressor

**Pull request closed without merging**